I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 46, in script
    label animate_text(who, text):
TypeError: missing a required argument: 'who'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 46, in script
    label animate_text(who, text):
  File "/home/temich719/renpy/renpy-8.2.1-sdk/renpy/ast.py", line 766, in execute
    values = apply_arguments(self.parameters, renpy.store._args, renpy.store._kwargs)
  File "/home/temich719/renpy/renpy-8.2.1-sdk/renpy/parameter.py", line 429, in apply_arguments
    return parameters.apply(args or (), kwargs or {}, ignore_errors)
  File "/home/temich719/renpy/renpy-8.2.1-sdk/renpy/parameter.py", line 283, in apply
    raise TypeError(msg) # from None
TypeError: missing a required argument: 'who'

Linux-6.7.7-200.fc39.x86_64-x86_64-with-glibc2.38 x86_64
Ren'Py 8.2.1.24030407
testNovel 1.0
Thu Mar 21 13:06:41 2024
